pywbem/_server.py

```diff
@@ -382,6 +382,11 @@ def create_namespace(self, namespace):
         update this WBEMServer object to reflect the new namespace
         there.
 
+        This method cannot create an Interop namespace because creating the
+        first Interop namespace with client operations depends on the prior
```

Comment:
Should probably remove first from "first interop namespace" since there can be only one interop namespace

Reply:
DONE